boxScale: 0.5
arrowScale: 1.0
boxRed: 1.0
boxGreen: 0.898
boxBlue: 0.039
boxAlpha: 0.5
name: 'BoxCounter'
description: 'Cone Counting Box'
headerID: 'velodyne'